### PR TITLE
Add an example to demonstrate long tuple cannot be printed

### PR DESCRIPTION
**Pull request by vishalsodani**
An attempt to fix the bug #589 .

### CODE REVIEW

File: src/primitives/tuples.md [hunk 1]
```diff
@@ -31,9 +31,14 @@ fn main() {
 
     // Tuples can be tuple members
     let tuple_of_tuples = ((1u8, 2u16, 2u32), (4u64, -1i8), -2i16);
-
+
```
Comment (steveklabnik) [inline src/primitives/tuples.md]: ✂️

**Comment by vishalsodani** [comment]
Why does this temporary restriction exists?

**Comment by steveklabnik** [comment]
> Why does this temporary restriction exists?

Because Rust doesn't have a way to write a function generically over integers, which would be needed to implement a trait for tuples of arbitrary lengths, which has an RFC accepted but not yet implemented.

(Changes look good!)

**Comment by srikwit** [comment]
We still don't have this implemented. Any suggestions on how this can be done?

**Pull request merged**